packages/usa-modal/src/test/template.html

```diff
@@ -44,8 +44,8 @@ <h2 class="usa-modal__heading" id="modal-sm-heading">
       </div>
     </div>
     <button type="button" id="close-button" class="usa-button usa-modal__close" aria-label="close this window" data-close-modal>
-      <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
-        <use xlink:href="{{ uswds.path }}/img/sprite.svg#close"></use>
+      <svg class="usa-icon" aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg">
+        <use href="{{ uswds.path }}/img/sprite.svg#close"></use>
```

Comment:
Does this html file need to have the svg link in it?

Reply:
@RachelCorsino no we can leave it out. This is for the unit test and we're not testing for this attribute in `modal.spec.js`.

Reply:
Resolved in [4dbefec](https://github.com/uswds/uswds/pull/6165/commits/4dbefecd5d272c477ae3908a5ee7c49f2657145f)